### DISCUSSION TITLE
No-Code Data-Literacy Tool

### DISCUSSION THREAD

**Discussion by digital-codes** [opensource9ja/danfojs#568] [Show and tell]
Hi there.
Working with open data and educational tools for data-literacy since quite some time I was always missing a tool which provides a low-barrier (=> no-code) entry into "data xyz" (analyzis, visualization, you name it ...). I worked with Orange3, Knime, Jupyter notebooks and some "free" tools from industry.  All of them require coding skills, registration, money etc 
When I came across Danfo.js I thought this might be a good way to build such a tool. And in fact, it looks very promising.
By chance, I've jsut seen today you had this idea in chapter8 of the book as well, but the code there doesn't compile and I think it is not very advanced.
So, I would appreciate comments on Daten.Cafe 
https://github.com/digital-codes/datencafe  
Keep in mind, this will be a non-commercial entry-level tool, but might still be interesting.
![datencafe](https://user-images.githubusercontent.com/34965919/233866964-47d002a5-7214-4dc9-b3d7-19d0d0bf9bdc.png)

**Comment by steveoni** [comment]
@digital-codes this looks awesome. Is it deployed for testing?

Also for the code in chapter 8 not compiling you can check this repo below re the compile issue might be as a result of Danfojs version

https://github.com/steveoni/Data-aRT

**Reply by digital-codes** [reply]
It runs at https://daten.cafe  (no sruprise). When you want to create a map, you have to tick the GDPR consent in the settings first (not documented yet). In the flow toolbar next to the ?  
Reason is Leaflet uses OSM tile server, which is not GDPR compliant. For workshops I have a very simple proxy for the tiles and for CORS requests to other resources. Available only when logged in....

**Reply by steveoni** [reply]
Will test it out once I’m on my system..

Awesome work

**Reply by digital-codes** [reply]
Curious to get feedback
